sample-apps/donet-sample-app/Controllers/MetricEmitter.cs

```diff
@@ -0,0 +1,137 @@
+using System;
+using System.Collections.Generic;
+using OpenTelemetry;
+using System.Diagnostics.Metrics;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Instrumentation;
+
+namespace donet_sample_app.Controllers
+{
+    public class MetricEmitter
+    {
+        const string DIMENSION_API_NAME = "apiName";
+        const string DIMENSION_STATUS_CODE = "statusCode";
+        
+
+        static string API_COUNTER_METRIC = "totalApiRequests";
```

Comment:
I don't think the metric names are in accordance with the spec: https://github.com/aws-observability/aws-otel-community/blob/master/sample-apps/SampleAppSpec.md?plain=1#L107

Reply:
Updated